components/payments/cmd/connectors/internal/connectors/atlar/currencies.go

```diff
@@ -5,5 +5,6 @@ import "github.com/formancehq/payments/cmd/connectors/internal/connectors/curren
 var (
 	supportedCurrenciesWithDecimal = map[string]int{
 		"EUR": currency.ISO4217Currencies["EUR"], //  Euro
+		"DKK": currency.ISO4217Currencies["DKK"],
```

Comment:
We need the `DKK` currency, because there is only one bank for the `EUR` currency in the Atlar default sandbox. After asking them for a second `EUR` bank, they asked us if we could use a `DKK` accounts for testing multi banking.